Resources/Prototypes/Research/industrial.yml

```
@@ -63,12 +63,13 @@
     state: building
   discipline: Industrial
   tier: 1
-  cost: 10000
+  cost: 12500 #CyberneticGenesis - (10000)
```

Старое значение можно не сохранять, его через историю можно посмотреть
```suggestion
  cost: 12500 # Adventure CyberneticGenesis
```